CONTRIBUTING.md

```diff
@@ -238,6 +238,15 @@ applying site config changes may require stopping and restarting this program.
 
 View your local documentation site at `localhost:8080/documentation/`
 
+## Updating Zoe Version and DocsUpdated
+
+In `[/.vuepress/config.js](/.vuepress/config.js)`, find the lines
+```
+zoeVersion: 'Alpha Release v0.8.1',
+zoeDocsUpdated: '2020-9-15'
+```
+Edit the values to be current.
```

Comment:
You might add that Zoe's version number is near the top of 

https://github.com/Agoric/agoric-sdk/tree/master/packages/zoe/package.json

(I was just cross-checking that 0.11.0 was still correct.)